lib/google_places/client.rb

```diff
@@ -148,7 +148,40 @@ def spot(place_id, options = {})
     def spots_by_query(query, options = {})
       Spot.list_by_query(query, @api_key, @options.merge(options))
     end
-
+    # Search for Spots within a give SW|NE bounds with query
+    #
+    # @return [Array<Spot>]
+    # @param [Hash] bounds
+    # @param [String] api_key the provided api key
+    # @param [Hash] options
+    # @option bounds [String, Integer] :se
+    #   the southeast lat|lng pair
+    # @option bounds [:se][String, Integer] :lat
+    #   the SE latitude
+    # @option bounds [:se][String, Integer] :lng
+    #   the SE longitude
+    # @option bounds [:se][String, Integer] :lat
+    #   the SE latitude
+    # @option bounds [:se][String, Integer] :lng
+    #   the SE longitude
+    # @option options [String,Array] :query
+    #   Restricts the results to Spots matching term(s) in the specified query
+    # @option options [String] :language
+    #   The language code, indicating in which language the results should be returned, if possible.
+    # @option options [String,Array<String>] :exclude ([])
+    #   A String or an Array of <b>types</b> to exclude from results
+    #
+    # @option options [Hash] :retry_options ({})
+    #   A Hash containing parameters for search retries
+    # @option options [Object] :retry_options[:status] ([])
+    # @option options [Integer] :retry_options[:max] (0) the maximum retries
+    # @option options [Integer] :retry_options[:delay] (5) the delay between each retry in seconds
+    #
+    # @see http://spreadsheets.google.com/pub?key=p9pdwsai2hDMsLkXsoM05KQ&gid=1 List of supported languages
```

Comment:
When opening this link I get an error: "We're sorry, this document is not published". Can you fix the link to a working version?